ansible/roles/rally/templates/create_and_bind_ports.json.j2

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
It's better to keep these task config files under [samples](https://github.com/huikang/ovn-scale-test/tree/ansible-docker-rally-port-binding/samples) folder.